charts/astronomer/templates/houston/houston-configmap.yaml

```diff
@@ -121,6 +129,54 @@ data:
           defaultUser:
             enabled: false
 
+          extraNetworkPolicies:
+          - namespaceSelector: {}
+            podSelector:
+              matchLabels:
+                tier: nginx
+                component: ingress-controller
+                release: {{ .Release.Name }}
+
+          webserverConfig: |
+            import os
+            from airflow import configuration as conf
+            from flask_appbuilder.security.manager import AUTH_REMOTE_USER
+            basedir = os.path.abspath(os.path.dirname(__file__))
+
+            # The SQLAlchemy connection string.
+            SQLALCHEMY_DATABASE_URI = conf.get('core', 'SQL_ALCHEMY_CONN')
+
+            # Flask-WTF flag for CSRF
+            CSRF_ENABLED = True
+
```

Comment:
```suggestion
```

This was already default, and this setting does nothing -- it should have been `WTF_CSRF_ENABLED`